src/Resources/contao/library/Contao/Search.php

```diff
@@ -298,6 +272,27 @@ public static function indexPage($arrData)
 		return true;
 	}
 
+	/**
+	 * @return string[]
+	 */
+	private function splitIntoWords(string $strText, string $strLocale): array
+	{
+		$iterator = \IntlBreakIterator::createWordInstance($strLocale);
+		$iterator->setText($strText);
+
+		$words = [];
```

Comment:
CS: use the long array syntax.

Reply:
Fixed in 8e546398628310081e5f8adfb32a58ccda76face